demo/bottom_up_img_demo.py

```diff
@@ -34,7 +34,7 @@ def main():
     parser.add_argument(
         '--kpt-thr', type=float, default=0.3, help='Keypoint score threshold')
     parser.add_argument(
-        '--oks-thr', type=float, default=0.9, help='NMS oks threshold')
+        '--pose-nms-thr', type=float, default=0.9, help='Pose nms threshold')
```

Comment:
I suggest we make it clear that pose-nms-thr is applied on OKS score in the help message.